slemicro/create_vm.sh

```diff
@@ -82,55 +91,73 @@ mkisofs -quiet -full-iso9660-filenames -o ${VMFOLDER}/ignition-and-combustion-${
 # Remove leftovers
 rm -Rf ${TMPDIR}
 
-# Create and launch the VM using UTM
-OUTPUT=$(
-osascript <<END
-tell application "UTM"
-	--- specify a boot ISO
-	set iso to POSIX file "${VMFOLDER}/ignition-and-combustion-${VMNAME}.iso"
-	-- specify the RAW file
-	set rawfile to POSIX file "${VMFOLDER}/${VMNAME}.raw"
-	--- create a new QEMU VM 
-	set vm to make new virtual machine with properties {backend:qemu, configuration:{cpu cores:${CPUS}, memory: ${MEMORY}, name:"${VMNAME}", architecture:"aarch64", drives:{{removable:true, source:iso}, {removable:false, source:rawfile}}}}
-	start vm
-	repeat
-		if status of vm is started then exit repeat
-	end repeat
-	get address of first serial port of vm
-end tell
+# if x86_64, convert the image to qcow2
+if [ $(uname -o) == "GNU/Linux" ]; then
```

Comment:
This should be Darwin first, elif GNU/Linux (to be consistent with the first `if`)